pkg/apis/dashboard/v2alpha1/register.go

```diff
@@ -32,7 +32,7 @@ var DashboardResourceInfo = utils.NewResourceInfo(GROUP, VERSION,
 				if dash != nil {
 					return []interface{}{
 						dash.Name,
-						dash.Spec.GetNestedString("title"),
+						dash.Spec.Title,
```

Comment:
@toddtreece  - what's the type of the `Spec` here? Think we should be ready to start generating go struct for v2 spec given the schema v2 is almost complete.